packages/core/src/__stampers__/api/web/stamper.ts

```diff
@@ -0,0 +1,218 @@
+import {
+  uint8ArrayToHexString,
+  stringToBase64urlString,
+  pointEncode,
+} from "@turnkey/encoding";
+import type { TStamp, ApiKeyStamperBase } from "@types";
+
+const DB_NAME = "TurnkeyStamperDB";
+const DB_STORE = "KeyStore";
+const stampHeaderName = "X-Stamp";
+
+function convertEcdsaIeee1363ToDer(ieee: Uint8Array): Uint8Array {
```

Comment:
feels like these two utility methods belong elsewhere (and also get unit tested)